code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -111,7 +111,6 @@
 	force = 10
 	caliber = "7.62mm"

```

Comment:
Caliber still doesn't seem to match, but I don't know what it actually does anymore.